MAiD - Case Management App/force-app/main/default/objects/Account/validationRules/profession_type.validationRule-meta.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>profession_type</fullName>
```

Comment:
@AmishMistri Why is full name field not having capital character? It should be Profession_Type